themes/ropensci/layouts/partials/comments.html

```diff
@@ -0,0 +1,38 @@
+<!-- discourse comments -->
+<!-- 
+group 1. posts before discourse came along - no discourse embed code
+group 2. posts that have topicid in their yaml header - get discourse embed code with topocid
+group 3. posts we're trying to automate embeds for, no topicid and newer than 2019
+  (to differentiate from older posts that also don't have topicid in their yaml)
+-->
+
+{{ with .topicid }}
```

Comment:
it's an useful pattern cf https://gohugo.io/templates/introduction/#example-1-with and https://gohugo.io/templates/introduction/#example-2-with--else

Reply:
nice